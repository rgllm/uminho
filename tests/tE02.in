["acaB","0 0 N","DDDDDS"]