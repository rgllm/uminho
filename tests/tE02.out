["<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>","<html xmlns='http://www.w3.org/1999/xhtml'>","<head>","<meta http-equiv='X-UA-Compatible' content='chrome=1' />","<meta http-equiv='Content-Type' content='text/html;charset=utf-8' />","<title>Tabuleiro Lightbot</title>","<script type='text/javascript' src='http://www.x3dom.org/release/x3dom.js'></script>","<link rel='stylesheet' type='text/css' href='style.css'/>","</head>","<body>","<center>","<h1>Tabuleiro Lightbot</h1>","</center>","<center>","<p class='case'>","<X3D xmlns='http://www.web3d.org/specifications/x3d-namespace' id='boxes' showStat='false' showLog='false'>","<scene>","<Viewpoint position='0.5 2.0 12' orientation='0 0 1 2' centerOfRotation='0.5 2.0 0' description='camera'></Viewpoint>","<shape DEF='quadrado'>","<Appearance>","<material diffuseColor='.3 .3 .3'/>","</Appearance>","<box size='.99 .99 .99'/>","</shape>","<shape DEF='quadradol'>","<appearance>","<material diffuseColor='0 0 1.0'/>","</appearance> <box size='.99 .99 .99'/>","</shape>","<transform DEF = 'robot'>","<transform scale='.5 .5 .5'>","<transform translation='0 0 1.4'  scale='0.8 0.7 .7'>","<shape>","<appearance>","<material diffuseColor='0.7 0.7 0.7'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","<transform translation='0 0 0' rotation='0 0 1 3.2' scale='1 1 1'>,","<shape>","<appearance>","<material diffuseColor='0.7 0.7 0.7'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","<transform translation='0 0.2 2.5' rotation='0 0 0 0' scale='.1 .1 .1'>","<shape>","<appearance>","<material diffuseColor='1 1 1'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","<transform translation='0 0.2 2.3' rotation='0 0 0 0' scale='0.03 0.03 0.3'>","<shape>","<appearance>","<material diffuseColor='0.7 0.7 0.7'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","<transform translation='-.3 .7 1.4' scale='.15 .05 .15'>","<shape>","<appearance>","<material diffuseColor='1 1 1'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","<transform translation='.3 .7 1.4' scale='.15 .05 .15'>","<shape>","<appearance>","<material diffuseColor='1 1 1'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","<transform translation='0 0.71 1.1' scale='.2 0 .075'>","<shape>","<appearance>","<material diffuseColor='1 1 1'/>","</appearance>","<sphere></sphere>","</shape>","</transform>","</transform>","</transform>","<transform translation='0 0 0'><shape USE='quadrado'/></transform>","<transform translation='0 1 2'><shape USE='quadrado'/></transform>","<transform translation='0 1 1'><shape USE='quadrado'/></transform>","<transform translation='0 1 0'><shape USE='quadrado'/></transform>","<transform translation='0 2 0'><shape USE='quadrado'/></transform>","<transform translation='0 3 1'><shape USE='quadradol'/></transform>","<transform translation='0 3 0'><shape USE='quadrado'/></transform>","<timeSensor DEF='time' cycleInterval='6' loop='true'> </timeSensor>","<PositionInterpolator DEF='move' key='0 0.2 0.4 0.6 0.8 1' keyValue='0 0 1  0 0 1  0 0 1  0 0 1  0 0 1  0 0 1  0 1 3 '> </PositionInterpolator>","<Route fromNode='time' fromField ='fraction_changed' toNode='move' toField='set_fraction'> </Route>","<Route fromNode='move' fromField ='value_changed' toNode='robot' toField='translation'> </Route>","<orientationInterpolator DEF='roda' key='0 0 0.16666666666666666 0.2 0.3666666666666667 0.4 0.5666666666666667 0.6 0.7666666666666666 0.8 0.9666666666666667 1' keyValue='0 0 1 1.57 0 0 1 1.57  0 0 1 0.0   0 0 1 0.0  0 0 1 -1.57   0 0 1 -1.57  0 0 1 -3.14   0 0 1 -3.14  0 0 1 -4.71   0 0 1 -4.71  0 0 1 -6.28   0 0 1 -6.28 '></orientationInterpolator>","<ROUTE fromNode='time' fromField='fraction_changed' toNode='roda' toField='set_fraction'></Route>","<ROUTE fromNode='roda' fromField='value_changed' toNode='robot' toField='set_rotation'></Route>","</scene> </X3D> </p> </body> </html>"]